packages/graphql/src/fragments/primitives.ts

```diff
@@ -1,16 +1,43 @@
 import type { FragmentOf } from 'gql.tada';
 import { graphql } from '../graphql';
 
+export const AppMetadata = graphql(`
+    fragment AppMetadata on AppMetadata {
+      __typename
+      description
+      developer
+      logo
+      name
+      platforms
+      privacyPolicy
+      termsOfService
+      url
+    }
+`);
+export type AppMetadata = FragmentOf<typeof AppMetadata>;
+
 export const App = graphql(
-  `fragment App on App {
-    address
-  }`,
+  `
+    fragment App on App {
+      __typename
+      address
+      graphAddress
+      sponsorshipAddress
+      defaultFeedAddress
+      namespaceAddress
+      treasuryAddress
+      createdAt
+      metadata {
+        ...AppMetadata
+      }
+    }
+  `,
+  [AppMetadata],
 );
 export type App = FragmentOf<typeof App>;
 
-export const Feed = graphql(
-  `fragment Feed on Feed {
-    address
-  }`,
-);
-export type Feed = FragmentOf<typeof App>;
+export const Feed = graphql(`
+    fragment Feed on Feed {
+      address
+    }
+`);
```

Comment:
Can we attach the beginning and ending of strings with backticks to the strings so to achieve some formatting consistency?

```
  `fragment AppMetadata on AppMetadata {
      __typename
      description
      developer
      logo
      name
      platforms
      privacyPolicy
      termsOfService
      url
    }`
    
    ...
    
  `fragment App on App {
      __typename
      address
      graphAddress
      sponsorshipAddress
      defaultFeedAddress
      namespaceAddress
      treasuryAddress
      createdAt
      metadata {
        ...AppMetadata
      }
    }`
    
    ...
    
    `fragment Feed on Feed {
      address
    }`
```